API/API.mli

```diff
@@ -4669,8 +4669,6 @@ sig
   val constant_has_body : Declarations.constant_body -> bool
   val is_opaque : Declarations.constant_body -> bool
   val eq_recarg : Declarations.recarg -> Declarations.recarg -> bool
-  val body_of_constant :
```

Comment:
Printing is another story, and there is no convenient API for this high-level feature yet. This is going to be part of the third round of the PR, when actually replacing abstract instances by another datastructure. There will definitely be a function to print a term in a context, just as we do using rel_contexts as of today.